pipedrive/__init__.py

```diff
@@ -35,7 +35,7 @@ def _request(self, endpoint, data, method='POST'):
             response, data = self.http.request(uri, method=method, headers={'Content-Type': 'application/x-www-form-urlencoded'})
         else:
             uri = PIPEDRIVE_API_URL + endpoint + '?api_token=' + str(self.api_token)
-            response, data = self.http.request(uri, method=method, body=urlencode(data), headers={'Content-Type': 'application/x-www-form-urlencoded'})
+            response, data = self.http.request(uri, method=method, body=json.dumps(data), headers={'Content-Type': 'application/json'})
```

Comment:
"We recommend using JSON body format when performing API requests" https://developers.pipedrive.com/docs/api/v1/#/

Reply:
We can see general issues with Pipedrive API server implementation (wrong specification of API server - they rely on "Content-Type" header instead of "Accept" - look to RFC: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept):
1) When using `requests` Python library, I can see HTML instead of JSON the same with curl
2) Documentation doesn't follow HTTP RFC in case of headers specyfication, what is wrong today (maybe 20 years ago it will be fine, but not in 2024!) -> https://pipedrive.readme.io/docs/core-api-concepts-requests

Sometimes junior developers needs to listen critical comments carefully :-) 
This is not comment to you @kevhill - this is for Pipedrive API server developers :-)